docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:centos5
+RUN rpm -Uvh http://dl.fedoraproject.org/pub/epel/5/x86_64/epel-release-5-4.noarch.rpm
+RUN yum install -y wget bzip2 bzip2-devel git gcc gcc-c++ patch zlib-devel make gcc44 gcc44-c++ cmake ncurses-devel
+RUN wget http://people.centos.org/tru/devtools-2/devtools-2.repo -O /etc/yum.repos.d/devtools-2.repo
+RUN yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-gcc-c++
+ENV MANPATH=""
+RUN source /opt/rh/devtoolset-2/enable
```

Comment:
These two lines should go in update_binstar_packages_docker_prebuilt.sh -- they turn on gcc-4.8 which is helpful for building a lot of packages. I'm not sure how well this gets cached between a recipe and a run.